tools/firmware-boot-sh.toit

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2024 Toitware ApS.
```

Comment:
Should the two licenses just be MIT? It makes sense to include it in full in the bash script.

Reply:
As discussed in person: outer license stays LPGL and inner one is MIT.
I added a comment that states that the inner license is MIT.